setup_LCG.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+case `uname` in
+  Linux) ECHO="echo -e" ;;
+  *) ECHO="echo" ;;
+esac
+
+usage(){
+	EXIT=$1
+	$ECHO "setup.sh [options]"
+	$ECHO
+	$ECHO "Options:"
+	$ECHO "-d              \tuse the developer branch of Coffea (default = 0)"
+	$ECHO "-h              \tprint this message and exit"
+	$ECHO "-n [NAME]       \toverride the name of the virtual environment (default = coffeaenv)"
+	exit $EXIT
+}
+
+NAME=coffeaenv
+LCG=/cvmfs/sft.cern.ch/lcg/views/LCG_101cuda/x86_64-centos7-gcc8-opt
```

Comment:
I think it would be better overall to have one setup script w/ singularity as the default and LCG as an alternative option.

Reply:
Yeah I agree. I was trying to keep the setup script short and sweet but I like the idea of one file to rule the all.